serveradmin/resources/management/commands/clear_migration_log.py

```diff
@@ -0,0 +1,49 @@
+"""Serveradmin - igvm_migration_log cleanup
+
+Copyright (c) 2020 InnoGames GmbH
+"""
+from django.core.management.base import BaseCommand
+from adminapi.dataset import Query
+from adminapi.filters import Not, Empty
+from datetime import datetime
+
+class Command(BaseCommand):
+    help = 'Check for old igvm_migration_log entries and deletes them'
+
+    def handle(self, *args, **options):
+        """Entry point for command
+
+        get all hypervisors which have an entry in igvm_migration_log
+        and delete all entries which are from the previous day.
+
+        """
+
+        all_hypervisor = Query({'igvm_migration_log': Not(Empty())},
+                           ['igvm_migration_log', 'hostname'])
+
+        for hypervisor in all_hypervisor:
+
+            for entry in hypervisor['igvm_migration_log']:
+                timestamp = entry.split(' ')[0]
```

Comment:
`split(' ')` will split on exactly ONE space. On default split will split like that:
```
split(sep=None, maxsplit=-1) method of builtins.str instance
    Return a list of the words in the string, using sep as the delimiter string.

    sep
      The delimiter according which to split the string.
      None (the default value) means split according to any whitespace,
      and discard empty strings from the result.
    maxsplit
      Maximum number of splits to do.
      -1 (the default value) means no limit.
```

To show it in action on an example:
```
>>> a = 'hello   h'
>>> a.split()
['hello', 'h']
>>> a.split(' ')
['hello', '', '', 'h']
>>>
```

The question is therefore, what's important to have explicitly `' '` and not the default behavior of split?

Reply:
Since the entry has excact one space, we can use the default behavior.